style.css

```diff
@@ -0,0 +1,852 @@
+*,
+html {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  padding: 30px 10px;
+  align-items: center;
+}
+
+header a {
+  font-family: "Roboto Serif", serif;
+  font-style: normal;
+  font-weight: 700px;
+  font-size: 18px;
+  line-height: 24px;
+  color: rgb(255, 255, 255);
+  text-decoration: none;
+}
+
+.home {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 20px;
+  gap: 12px;
+  width: 100%;
+  height: auto;
+  background: #1c1a19;
+  background-image: url(images/Group.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+}
```

Comment:
- [ ] As requested by the previous code reviewers, your project should be responsive on all screen sizes but currently, you have extra white spaces at the right at >768px screen sizes. as you can see in the video below 👇 Make sure you adjust the widths and align containers correctly to avoid this to make your project responsive.

https://user-images.githubusercontent.com/69077061/236640567-859600fb-830b-4e1e-90e1-8bacf2056485.mp4